main.rb

```diff
@@ -38,7 +37,7 @@ def main
         puts "Sorry none of you won"
         on_game = false
       else
-        game.change_player
+        game.switch_players
       end
```

Comment:
Same as Player constants. even though the `Board` class is not directly required in your `main` method, `main` is aware that a class named `Board` exists and that it has `constants`. This makes `main` susceptible to being broken if `board` is modified for any reason. 

We can break this problem into sub problems.
1. You want to know if the game is still active. You could know that in main by asking `Game`. Something like `Game.still_active?`
2. You want to know how the game ended so you put the appropiate message. They way I see this is:
 - You ask if the game is still active
- If the game is not active anymore that means either there is a sequence or the board is full. How can `game ` return that information to `main` so that `main` can take the appropiate decision (`main` asks: what do I print?)